src/Agent.Sdk/Util/VssUtil.cs

```diff
@@ -27,6 +27,13 @@ public static void InitializeVssClientSettings(ProductInfoHeaderValue additional
 
             VssClientHttpRequestSettings.Default.UserAgent = headerValues;
             VssClientHttpRequestSettings.Default.ClientCertificateManager = clientCert;
+#if OS_LINUX || OS_OSX
+            // The .NET Core 2.1 runtime switched its HTTP default from HTTP 1.1 to HTTP 2.
+            // This causes problems with some versions of the Curl handler.
+            // See GitHub issue https://github.com/dotnet/corefx/issues/32376
+            VssClientHttpRequestSettings.Default.UseHttp11 = true;
```

Comment:
search for `InitializeVssConnection`, there might be 2 more places we need to change.

Reply:
You were right. I updated those two places to also specify `UseHttp11`